internal/codespaces/grpctunnel/rwio.go

```diff
@@ -0,0 +1,23 @@
+// Package grpctunnel provides utilities for connecting to a codespace through a gRPC tunnel.
+package grpctunnel
+
+import "io"
+
+// rwio is an utility to combine the closing of a R/W interface on a single operation.
+type rwio struct {
+	io.ReadCloser
+	io.WriteCloser
+}
```

Comment:
This struct is extracted from the ssh command. It is used for the `gh cs ssh --stdio` option.